test/FunctionalTests/Microsoft.OData.Core.Tests/UriParser/Parsers/UriQueryExpressionParserTests.cs

```diff
@@ -165,6 +165,41 @@ public void ParseApplyWithMultipleAggregateExpressionsShouldReturnAggregateToken
         }
 
         [Fact]
+        public void ParseApplyWithSingleCountExpressionShouldReturnAggregateToken()
+        {
+            var apply = "aggregate($count as Count)";
+
+            var actual = this.testSubject.ParseApply(apply);
+            actual.Should().NotBeNull();
+            actual.Should().HaveCount(1);
+
+            var aggregate = actual.First() as AggregateToken;
+            aggregate.Should().NotBeNull();
+            aggregate.Expressions.Should().HaveCount(1);
+
+            VerifyAggregateExpressionToken("$count", AggregationMethod.VirtualPropertyCount, "Count", aggregate.Expressions.First());
+        }
+

```

Comment:
A negative test would be nice, something like "aggregate(#count with sum as Count)" and possible "aggregate(#count). 

From the spec:
"It MUST always specify an alias and MUST NOT specify an aggregation method."